src/style.css

```diff
@@ -1,3 +1,112 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
 body {
-    background-color: bisque;
-}
\ No newline at end of file
+  height: 100vh;
+  width: 100%;
+  background: #dfdad9;
+  font-family: 'poppins', Arial, Helvetica, sans-serif;
+  display: flex;
+}
```

Comment:
- [ ] Kindly style the application according to the template used in this [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg)

### Template 

![image](https://user-images.githubusercontent.com/77106962/165410580-28889be2-a1ef-4e35-be36-ef168aa9e768.png)

### Your Design

![image](https://user-images.githubusercontent.com/77106962/165410629-7c96a18c-97b5-4150-8e71-eb9f2a0e95f3.png)